e2e/pages/import.po/index.ts

```diff
@@ -70,17 +82,28 @@ export class Import extends BasePage {
     return elements.unitTestDescriptionSwitch.switchOn();
   }
 
+  switchOnIntoLastTestRun() {
+    return elements.lastTestRunSwitch.switchOn();
+  }
+
   uploadFile(absolutePath: string) {
     return elements.fileUpload.sendKeys(absolutePath);
   }
 
-  async waitForImportResultsCountToBe(count: number) {
-    const oldLatestTestRunId: number = await this.getLatestImportedTestRunId();
-    return browser.wait(async () => {
+  async getTestRunIdFromImportRow(rowIndex: number) {
+    return +(await elements.importResultsTable.getCellTextUsingRowIndex(importResultColumns.testRunId, rowIndex));
+  }
+
+  async waitForNewImportResult(lastTestRunDate: Date) {
+    return waiter.forTrue(async () => {
       await elements.importResultsTable.clickRefresh();
-      const actualLatestTestRunId = await this.getLatestImportedTestRunId();
-      const isFinished = await this.isLatestImportFinished();
-      return oldLatestTestRunId !== actualLatestTestRunId && isFinished;
-    }, browser.allScriptsTimeout).catch(() => false);
+      try {
+        const actualLatestTestRundate = await this.getLatestImportedTestRunDate();
+        const isFinished = await this.isLatestImportFinished();
+        return lastTestRunDate < actualLatestTestRundate && isFinished;
+      } catch (error) {
+        return false;
+      }
+    }, 5, 3000);
```

Comment:
is it possible to move timeouts to config file?

Reply:
this one is ralated to only import time, not sure if I want to store in separately from import page object